lib/CampMinder/SignedRequestFactory.rb

```diff
@@ -17,15 +13,15 @@ def is_valid_request?(signed_request)
   end
 
   def get_payload(signed_payload)
-    Base64.urlsafe_decode64(signed_payload.split(".").last)
+    CampMinder::Base64.urlsafe_decode64(signed_payload.split(".").last)
   end
 
   def encode_signature(encoded_payload)
-    Base64.urlsafe_encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new("sha256"), @secret_code, encoded_payload)).strip()
+    CampMinder::Base64.urlsafe_encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new("sha256"), @secret_code, encoded_payload)).strip()
```

Comment:
Line is too long. [131/80]<br>Do not use parentheses for method calls with no arguments.